Tests/LoggerTests/Loggers/FileLoggerTests.swift

```diff
@@ -449,6 +449,60 @@ class FileLoggerTests: XCTestCase {
         XCTAssertEqual(fileLogs[3].message.description, "[🚗] Some message")
         XCTAssertEqual(fileLogs[4].message.description, encodedCodableString)
     }
+    
+    func test_delete_log_files() throws {
```

Comment:
Maybe more descriptive name for the function? Something like `test_deleting_log_files_resets_the_logger_configuration`...